src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java

```diff
@@ -24,10 +24,10 @@ public class MoveToPos extends SwerveCommand {
     public static final double translationKp = 2;
     public static final double translationKi = 0;
     public static final double translationKd = 0;
-    
+
     private double tolerance = 0.05;
     private double rotationTolerance = 4;
-    private boolean debugDash = false;
+    private boolean debugDash = true;
```

Comment:
Probably this and all other debug functions should be removed or be in the dashboard subsystem (I think its better to remove)